### DISCUSSION TITLE
Expose Variant Selection in the Outliner

### DISCUSSION THREAD

**Discussion by mdubrois** [Autodesk/maya-usd#2449] [Q&A]
Hi everyone,

There is a way or a plan to expose the current variant selection of a primitive in the Outliner.
I mean something like that for a variant selection rep:usd and geo:main
![maya_outliner_expose_variant_selection](https://user-images.githubusercontent.com/2589772/176617363-4a5f400b-dcca-48b9-a0a6-98f5f5284e56.png)

For your interest it's did like that in Houdini
![houdini_scene_graph_tree_expose_variant_selection](https://user-images.githubusercontent.com/2589772/176617480-90d9cc05-f103-48df-bdac-1f205bd137e3.png)

Our goal is to allow the artists to quickly know which variants are currently in use.

Thanks

**Comment by womanyoyoyo-adsk** [comment]
Hi @mdubrois,

Where this sits in our plan is we are researching scenarios to start design on the way variants are created, switched and modified. This is insightful that you need this. I'll post designs on variants when we have them.

Thanks,
Natalia (designer on MayaUSD)
